src/clj_pgp/core.clj

```diff
@@ -255,18 +255,21 @@
 
 ;; ## PGP Object Decoding
 
+(defn ^:no-doc read-next-object [factory]
```

Comment:
Please maintain the style here, so arg vector should go on a new line. A docstring would be helpful as well.